lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_html.inc

```diff
@@ -30,6 +30,8 @@ function paraneue_dosomething_preprocess_html(&$vars) {
 
   $vars['use_facebook_pixel'] = theme_get_setting('use_facebook_pixel');
 
+  $vars['use_google_tag_manager'] = theme_get_setting('use_google_tag_manager');
+
   $vars['enable_ad_tracking'] = variable_get('dosomething_setting_ad_tracking_enabled', FALSE);
```

Comment:
Should we just hard-code this one to `FALSE`?

Reply:
Oh yeah, probs. So that we don't have to bother removing all the code for it elsewhere and just make sure it's not added.